_posts/2020-09-28-biased-locking-help.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: Biased locking removed from Java - does it affect you ?
+date: 2020-09-28
+tags: help
+synopsis: In Java 15 *biased locking* is removed from the Java Virtual Machine as per JEP 374. We need your help seeing if it affects Quarkus performance in your usecase.
```

Comment:
AIUI in JDK 15 it is no longer the *default* in JDK 15 and might be removed in 16 or 17.

Reply:
Yes in jdk15 the default has been reset to disabled. It can still be enabled. This should say something like

`In Java 15 *biased locking* has been deprecrated as per JEP 374. This means the default setting has been switched from enabled to disabled and the plan is to remove support for it from the Java Virtual Machine in a pending release.`

Reply:
ive pushed a new version - let me know if that works for you.